### PR TITLE
Fix for #181

### PR DESCRIPTION
**Pull request by levesswe**
Blip: Re-wrote the logic of this script as the previous version was prone to skipping matches.  Also modified script to produce spreadsheets for user reference.

### CODE REVIEW

File: bulk/review-qw-info.vbs [hunk 1]
```diff
@@ -40,154 +40,465 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("11/29/2016", "Re-wrote logic as previous version was prone to skipping matches, see issue #181.", "Wendy LeVesseur, Anoka County")
```
Comment (theVKC) [inline bulk/review-qw-info.vbs]: This description isn't entirely helpful for non-scriptwriters. Issue numbers and the phrase "re-wrote logic" isn't super helpful if you aren't a scriptwriter. Can you rewrite this to be simpler to understand for non-techies?

File: bulk/review-qw-info.vbs [hunk 2]
```diff
@@ -40,154 +40,465 @@ changelog = array()
...
-			placeholder_qw_string = ""
+cases_array_ncp = trim(cases_array_ncp)
+cases_array_ncp = split(cases_array_ncp, " ")
 
+number_of_cases = ubound(cases_array_ncp)
+DIM ncp_array()
+ReDim ncp_array(number_of_cases, 6)
 
-			DO
+'>>>> HERE ARE THE 6 POSITIONS WITHIN THE ARRAY <<<<
```
Comment (theVKC) [inline bulk/review-qw-info.vbs]: Nice comments here!

File: bulk/review-qw-info.vbs (hunk 1 above)
Comment (theVKC) [inline bulk/review-qw-info.vbs]: Oooh, also, make sure the date is correct. :)

Comment (levesswe) [inline bulk/review-qw-info.vbs]: What language do you want? That it adds the spreadsheet to show what the script did?  It's hard to think like a non-script writer.

Comment (theVKC) [inline bulk/review-qw-info.vbs]: This is a bit subjective. Maybe something like "Bugs with catching all matches have been resolved, and new functionality to show a spreadsheet during the process has been added."

Sometimes for me, it helps to explain it to a non-scriptwriter and ask them to describe it.

File: bulk/review-qw-info.vbs [hunk 3]
```diff
@@ -40,7 +40,7 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
-call changelog_update("11/29/2016", "Re-wrote logic as previous version was prone to skipping matches, see issue #181.", "Wendy LeVesseur, Anoka County")
+call changelog_update("12/06/2016", "Fixed the skipping matches issue, and added spreadsheets for user reference.", "Wendy LeVesseur, Anoka County")
```
Comment (theVKC) [inline bulk/review-qw-info.vbs]: Looks great! Thanks for doing it!

**Pull request merged**